src/smirnoffio.py

```diff
@@ -667,6 +653,55 @@ def build_system_mval_masks(self):
         logger.info("system_mval_masks is built for faster gradient evaluations")
         self.system_mval_masks = system_mval_masks
 
+class TorsionProfileTarget_SMIRNOFF(TorsionProfileTarget):
+    """ Force and energy matching using SMIRKS native Open Force Field (SMIRNOFF). """
+    def __init__(self,options,tgt_opts,forcefield):
+        ## Default file names for coordinates and key file.
+        self.set_option(tgt_opts,'pdb',default="conf.pdb")
```

Comment:
If this PDB is only being created for the purpose of loading coordinates using the OFF toolkit, it may be possible to replace it with coordinates loaded from mol2 (as long as only a single molecule is being loaded at a time)

Reply:
The PDB file is also used for adding fields to the `forcebalance.molecule` object which is used through FB in several places, so I think it's best to keep it.